src/core/TypescriptConfig.ts

```diff
@@ -110,8 +110,8 @@ export class TypescriptConfig {
 				const configFileRelPath = this.configFile.replace(this.context.appRoot, "");
 				this.context.log.echoInfo(`Typescript config file:  ${configFileRelPath}`);
 				configFileFound = true;
-				const res = ts.readConfigFile(this.configFile, p => fs.readFileSync(p).toString());
-				config = res.config;
+				const res = ts.readConfigFile(this.configFile, ts.sys.readFile);
+				config = { ...res.config, compilerOptions: ts.parseJsonConfigFileContent(res.config, ts.sys, this.context.appRoot).options };
 				if (res.error) {
 					this.context.log.echoError(`Errors in ${configFileRelPath}`);
```

Comment:
Trying to understand how the code works here.

If there are errors, the current code just echo and continue processing.

Is this desirable? I'll need to load the files recursively, so I need to fail the call if one of the files can't be read.